libs/providers/flagd/project.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/providers/flagd/src",
+  "projectType": "library",
+  "targets": {
+    "publish": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "command": "npm run publish-if-not-exists",
+        "cwd": "dist/libs/providers/flagd"
+      },
+      "dependsOn": [
+        {
+          "projects": "self",
+          "target": "package"
+        }
+      ]
+    },
+    "generate": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "commands": [
+          "git submodule update --init --recursive",
+          "rm -f -r ./src/proto",
+          "cd schemas && buf generate buf.build/open-feature/flagd --template protobuf/buf.gen.ts.yaml",
+          "mv -v ./proto ./src"
+        ],
+        "cwd": "libs/providers/flagd",
+        "parallel": false
+      }
+    },
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["libs/providers/flagd/**/*.ts"]
+      },
+      "dependsOn": [
+        {
+          "projects": "self",
+          "target": "generate"
+        }
+      ]
+    },
+    "test": {
+      "executor": "@nrwl/jest:jest",
+      "outputs": ["coverage/libs/providers/flagd"],
+      "options": {
+        "jestConfig": "libs/providers/flagd/jest.config.ts",
+        "passWithNoTests": true
+      },
+      "dependsOn": [
+        {
+          "projects": "self",
+          "target": "generate"
+        }
+      ]
+    },
+    "package": {
+      "executor": "@nrwl/web:rollup",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "project": "libs/providers/flagd/package.json",
+        "outputPath": "dist/libs/providers/flagd",
+        "entryFile": "libs/providers/flagd/src/index.ts",
+        "tsConfig": "libs/providers/flagd/tsconfig.lib.json",
+        "compiler": "babel",
```

Comment:
You may want to consider adding ` "buildableProjectDepsInPackageJsonType": "dependencies"`. NX will automatically add dependencies to the packaged project.json. You can then explicitly set peer dependencies in the package.json before hand (which you're already doing).

Reply:
added